tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -404,6 +433,34 @@ def create_rke_cluster_config(aws_nodes):
     return clusterfilepath
 
 
+def create_aks_cluster():
+    tf_dir = DATA_SUBDIR + "/" + "terraform/aks"
+    aks_k8_s_version = os.environ.get('RANCHER_AKS_K8S_VERSION', '')
+    aks_location = os.environ.get('RANCHER_AKS_LOCATION', '')
+    client_id = os.environ.get('ARM_CLIENT_ID', '')
+    client_secret = os.environ.get('ARM_CLIENT_SECRET', '')
```

Comment:
We don't need tenant ID and Subscription ID here to create the cluster?

Reply:
That's loaded from the ARM variables and used by the azure cli client.
I use these two to pass these values to the cluster creation https://github.com/rancher/rancher/pull/30987/files#diff-eb90b3de3553e328d8a4b6c4e0fbbf42db2a2612ef538c5bf9399dfd2ae8c883R37